### PR TITLE
nikolai.berestevich: Blog : articles from cms now

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-   blog/articles : articles from cms now

## Type of change

-   [ ] Bug fix
-   [ x ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/blog/articles/_article-card.js [hunk 1]
```diff
@@ -65,18 +65,23 @@ const ArticleCard = ({ item }) => {
         <RedirectLink to={`/blog/articles/${item.slug}`}>
             <ArticleCardWrapper>
                 <ImageWrapper>
-                    <ContainedImg src={item.image} alt="Video card" width="100%" />
+                    <QueryImage
+                        data={item.main_image.imageFile.childImageSharp.gatsbyImageData}
```
Comment (sean-binary) [inline src/pages/blog/articles/_article-card.js:69]:
```suggestion
                        data={item.main_image}
```

Comment (sean-binary) [reply src/pages/blog/articles/_article-card.js:69]: This should be sufficient as we use `getImage` inside QueryImage component.

File: src/pages/blog/articles/_first-article.js [hunk 2]
```diff
@@ -57,31 +56,28 @@ const RedirectLink = styled(LocalizedLink)`
...
     return (
-        <RedirectLink to={`/blog/articles/${article_data[0].slug}/`}>
+        <RedirectLink to={`/blog/articles/${item.slug}/`}>
             <StyledFlex jc="flex-start" mt="96px">
                 <ImageWrapper>
-                    <CoverImg
-                        src="https://source.unsplash.com/random/10"
-                        alt="Video card"
+                    <QueryImage
+                        data={getImage(item.main_image.imageFile)}
```
Comment (sean-binary) [inline src/pages/blog/articles/_first-article.js:65]:
```suggestion
                        data={item.main_image.imageFile}
```

Comment (sean-binary) [reply src/pages/blog/articles/_first-article.js:65]: We don't need this as QueryImage already has `getImage`.

File: src/pages/blog/articles/_all-articles.js [hunk 3]
```diff
@@ -17,9 +17,9 @@ const AllArticles = ({ article_data }) => {
                 <StyledImg src={RightArrow} height="16" width="16" />
                 <Text>All articles</Text>
             </Flex>
-            <FeaturedArticle article_data={article_data} />
+            <FirstArticle item={article_data[0]} />
             <VideoGrid m="40px 0">
-                {article_data.map((item) => {
+                {article_data.slice(1).map((item) => {
```
Comment (kevinw-binary) [inline src/pages/blog/articles/_all-articles.js:22]: Do we need to sort the articles by published date ?

Comment (sean-binary) [reply src/pages/blog/articles/_all-articles.js:22]: Yep that's right, we do.

Comment (nikolaiberestevich-deriv) [reply src/pages/blog/articles/_all-articles.js:23]: done in the next PR

File: src/pages/blog/articles/_all-articles.js [hunk 4]
```diff
@@ -2,12 +2,13 @@ import React from 'react'
...
 import { StyledImg, Container, VideoGrid } from '../common/_styles'
 import ArticleCard from './_article-card'
-import FeaturedArticle from './_featured-article'
+import FirstArticle from './_first-article'
 import { Flex } from 'components/containers'
 import { Text, LocalizedLinkText } from 'components/elements'
 import RightArrow from 'images/svg/black-right-arrow.svg'
 
 const AllArticles = ({ article_data }) => {
+    console.log(article_data)
```
[the remark this answers is not in the document]
Comment (nikolaiberestevich-deriv) [reply src/pages/blog/articles/_all-articles.js:11]: oO

**Review by sean-binary — APPROVED** [review]

**Pull request merged**